package.json

```diff
@@ -82,6 +82,7 @@
     "karma-failed-reporter": "^0.0.3",
     "karma-firefox-launcher": "^1.0.1",
     "karma-jasmine": "^1.1.0",
+    "karma-junit-reporter": "^1.2.0",
```

Comment:
when you npm install, you would see `package-lock.json` getting updated. Can you check in that as well.

Reply:
done. https://github.com/elastic/apm-agent-rum-js/pull/323/commits/ebef14e8efb37b3f80a74e610c04f4d702442581